### DISCUSSION TITLE
🧩 FEATURE REQUEST:  In the app there is no place to see why you reputation has increased or decreased.

### DISCUSSION THREAD

**Discussion by msamgan** [dailydotdev/daily#1299] [Feature Requests]
### Concise Description

In the app there is no place to see why you reputation has increased or decreased.

### Description

In the app there is no place to see why you reputation has increased or decreased. There should be a profile section where I can track my reputation history. Which will give me a better Idea about what to do or not to do for the sake of my reputation. 

### Solution Proposed

A section in my profile with reputation history.

<img width="597" alt="image" src="https://github.com/user-attachments/assets/ad526cea-bce3-425a-8402-b4a8178eb956">

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @msamgan,

Thanks for the suggestion :)

**Reply by msamgan** [reply]
you are welcome, I would like to help in building this feature if a big guidance is provided. Where to get started and all.
